docs/developer/admin-rest-api/data-volumes.md

```diff
@@ -99,10 +99,20 @@ This section outlines the procedure for creating, sharing, and submitting (Resea
 !!! Note
     Data volume admins can create data volumes within specific projects. Since data volumes are created from PVCs, there has to be a PVC in the namespace of a run:ai project for Run:Ai to have access to it and create the Data volume from it. Once the DV is created, the admin manages its sharing configurations.
```

Comment:
Data volumes are created from PVs

Reply:
@alonlavian This was from your doc. Is there a difference betwen a PV and PVC? A persisten volume, vs a persistent volume claim?

Reply:
yes. Daniel corrected me that PV is the right term here

Reply:
```suggestion
    Data volume admins can create data volumes within specific projects. Since data volumes are created from PVs, there has to be a PV in the namespace of a run:ai project for Run:Ai to have access to it and create the Data volume from it. Once the DV is created, the admin manages its sharing configurations.
```